models/category/src/category-draft/presets/sample-data-fashion/bottoms-kids.ts

```diff
@@ -0,0 +1,16 @@
+import { LocalizedString, Reference } from '@commercetools-test-data/commons';
+import * as CategoryDraft from '../..';
+import { TCategoryDraftBuilder } from '../../../types';
+
+const bottomsKids = (): TCategoryDraftBuilder =>
+  CategoryDraft.presets
+    .empty()
+    .name(LocalizedString.presets.empty()['en-US']('Bottoms'))
+    .slug(LocalizedString.presets.empty()['en-US']('bottoms_kids'))
+    .description(LocalizedString.presets.empty()['en-US']('Bottoms'))
+    .parent(Reference.presets.category()['key']('clothing-kids'))
```

Comment:
Here and elsewhere...
```suggestion
    .parent(Reference.presets.category().key('clothing-kids'))
```

Reply:
The parent keys should be pulled in from the appropriate preset and referenced as variables so there aren't any magic strings hanging around.